lib/utils/resolveObject.js

```diff
@@ -55,7 +55,7 @@ function traverseObj(obj) {
   var key;
 
   for (key in obj) {
-    // We want to check for ignoredKeys, this is so
+    // We want to check for ignoredKeys, this is to
     // we can skip over attributes that should not be
```

Comment:
// we can skip over attributes that should not be

// skip over attributes that should not be